### DISCUSSION TITLE
Editor – provide preset aspect ratios for cropping

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#898] [Enhancements]
**Screenshot from device gallery app**
![image](https://github.com/ente-io/photos-app/assets/1161789/4068de6d-6ef5-4e88-8b40-8cc13b8e0fb0)

**Comment by cpaulik** [comment]
If you want to start smaller - for me the most important things would be:

1. Crop keeping aspect ratio of original photo
2. Keep aspect ratio of original photo but flip orientation

Very seldom do I use other aspect ratios - that's just me of course.

**Reply by cpaulik** [reply]
Just found a workaround for use case 1.

If you just use two fingers to zoom into the photo it will keep the aspect ratio.
That's actually quite nice. Discovered it when looking at the [example page of the component](https://pub.dev/packages/image_editor)
